api/python/t4/packages.py

```diff
@@ -424,8 +424,7 @@ def load(cls, readable_file):
         """
         reader = jsonlines.Reader(readable_file)
         meta = reader.read()
-        # Pop the top hash -- it should only be calculated dynamically
-        meta.pop('top_hash', None)
+        meta.pop('top_hash', None)  # Obsolete
```

Comment:
i guess we're keeping this in for old manifests? i would note as of which PR this becomes obsolete